diff --git a/node_modules/@mikro-orm/core/metadata/MetadataValidator.js b/node_modules/@mikro-orm/core/metadata/MetadataValidator.js
index 19fe418..4ff2dc5 100644
--- a/node_modules/@mikro-orm/core/metadata/MetadataValidator.js
+++ b/node_modules/@mikro-orm/core/metadata/MetadataValidator.js
@@ -35,10 +35,10 @@ class MetadataValidator {
         if (discovered.length === 0 && warnWhenNoEntities) {
             throw errors_1.MetadataError.noEntityDiscovered();
         }
-        const duplicates = utils_1.Utils.findDuplicates(discovered.map(meta => meta.className));
-        if (duplicates.length > 0) {
-            throw errors_1.MetadataError.duplicateEntityDiscovered(duplicates);
-        }
+        // const duplicates = utils_1.Utils.findDuplicates(discovered.map(meta => meta.className));
+        // if (duplicates.length > 0) {
+        //     throw errors_1.MetadataError.duplicateEntityDiscovered(duplicates);
+        // }
         // validate we found at least one entity (not just abstract/base entities)
         if (discovered.filter(meta => meta.name).length === 0 && warnWhenNoEntities) {
             throw errors_1.MetadataError.onlyAbstractEntitiesDiscovered();
