diff --git a/node_modules/@polkadot/util/logger.cjs b/node_modules/@polkadot/util/logger.cjs
index bc3cb92..f24cce9 100644
--- a/node_modules/@polkadot/util/logger.cjs
+++ b/node_modules/@polkadot/util/logger.cjs
@@ -77,8 +77,6 @@ function apply(log, type, values) {
     const fnResult = values[0]();
     return apply(log, type, Array.isArray(fnResult) ? fnResult : [fnResult], maxSize);
   }
-
-  console[logTo[log]]((0, _formatDate.formatDate)(new Date()), type, ...values.map(loggerFormat).map(formatWithLength(maxSize)));
 }
 
 function noop() {// noop
